### PR TITLE
Final code merged

### PR DESCRIPTION
**Pull request by shindesimantini6**
The animation, charts, classes all have been merged. you only need to run final_shopper.py to do the following sequence -->
1. opens a CSV
2. runs a while loop
   a. adds customers
   b. adds minute and find next state
   c. Prints it and appends to the CSV

   e. returns a current time
3. creates chart for each timestamp
4. creates a frame with the chart
5. shows the frame
6. removes customers already in checkout.

In final_shopper.py the user has the option to select start and end times, lambda for Poisson and Supermarket name.

In addition the merge request contains:
1.Customer class
2. Supermarket class
3. SupermarketMap class
4. create charts function
5. required images
6. required input data
7. README.md

Please have a look at all the files, and comment. And when we all approve lets merge this into the main branch. 

\rfc @CrisVillatoro  @sirvictorson

### CODE REVIEW

File: final_shopper.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+#D..DS..SZ..ZV..V#
+#D..DS..SZ..ZV..V#
+#D..DS..SZ..ZV..V#
+#D..DS..SZ..ZV..V#
+#D..DS..SZ..ZV..V#
+##...............#
+##..CC..CC..CC...#
+##..CC..CC..CC...#
+##...............#
+##XX##########EE##
```
Comment (CrisVillatoro) [inline final_shopper.py:25]: We have to change some of the positions of the icons

File: SupermarketMap.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            return self.extract_tile(3, 13)
+        elif char == "E":
+            return self.extract_tile(7, 3)
+        elif char == "S":
+            return self.extract_tile(5, 8)     
+        elif char == "V":
+            return self.extract_tile(2, 11)        
+        elif char == "X":
+            return self.extract_tile(6, 10)
+        else:
```
Comment (CrisVillatoro) [inline SupermarketMap.py:113]: The icon of the titles assigned to E, X and C have to checked. 
Coordinates for E (8,3)
Coordinates for X (8,14)
Coordinates for C (8,3)

File: final_shopper.py (hunk 1 above)
Comment (CrisVillatoro) [inline final_shopper.py:25]: The right position is #########XX###EE##

**Review by CrisVillatoro — CHANGES_REQUESTED** [review]

**Review by CrisVillatoro — APPROVED** [review]

**Review by sirvictorson — COMMENTED** [review]
Everything is fine

**Pull request merged**